### PR TITLE
added basic controller for CURD artical resource

### PR DESCRIPTION
**Pull request by joney000**
Starting with base controllers and views for Create , Update , Read List of Articles , delete Articles .

### CODE REVIEW

File: app/controllers/articles_controller.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
+class ArticlesController < ApplicationController
+  def index
+    @articles = Article.all
+  end
```
Comment (tarunjadhwani) [inline app/controllers/articles_controller.rb:4]: leave a line after every def.

File: app/controllers/articles_controller.rb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  def new
+    @article = Article.new
+  end
+  def show
+    @article = Article.find(params[:id])
+  end
+  def create
+    @article = Article.new(article_params)
+    if @article.save
```
Comment (tarunjadhwani) [inline app/controllers/articles_controller.rb]: use flashes here to convey if the article was created.

File: app/controllers/articles_controller.rb [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    if @article.update(article_params)
+      redirect_to @article
+    else
+      render 'edit'
+    end
+  end
+  def destroy
+    @article = Article.find(params[:id])
+    @article.destroy
+    redirect_to articles_path
```
Comment (tarunjadhwani) [inline app/controllers/articles_controller.rb]: same here. Flash messages.

File: app/controllers/comments_controller.rb [hunk 4]
```diff
@@ -0,0 +1,17 @@
+class CommentsController < ApplicationController
+  def create
+    @article = Article.find(params[:article_id])
+    @comment = @article.comments.create(comment_params)
+    redirect_to article_path(@article)
+  end
+  def destroy
+    @article = Article.find(params[:article_id])
+    @comment = @article.comments.find(params[:id])
```
Comment (tarunjadhwani) [inline app/controllers/comments_controller.rb]: Don't need to find article here. just send comment_id and delete it.

File: app/models/comment.rb [hunk 5]
```diff
@@ -0,0 +1,3 @@
+class Comment < ActiveRecord::Base
+  belongs_to :article
```
Comment (tarunjadhwani) [inline app/models/comment.rb:2]: add validation.

**Comment by tarunjadhwani** [comment]
@joney000 Secrets.yml should not be committed to git.

**Comment by joney000** [comment]
Secrets.yml removed from all history and added to git ignore

File: app/controllers/articles_controller.rb (hunk 1 above)
Comment (joney000) [inline app/controllers/articles_controller.rb:4]: done.

File: app/controllers/articles_controller.rb (hunk 2 above)
Comment (joney000) [inline app/controllers/articles_controller.rb]: added flashes

File: app/controllers/comments_controller.rb (hunk 4 above)
Comment (joney000) [inline app/controllers/comments_controller.rb]: corrected it . removed Artical.find query in destroy.

File: app/models/comment.rb (hunk 5 above)
Comment (joney000) [inline app/models/comment.rb:2]: added validation in comments as well as artical .

**Comment by tarunjadhwani** [comment]
@joney000 remove erb templates and then you can merge this PR

**Pull request merged**